bundles/org.openhab.binding.chatgpt/README.md

```diff
@@ -56,16 +56,15 @@ Each channel of type `chat` takes the following configuration parameters:
 | topP          | decimal | A value between 0 and 1. An alternative to sampling with temperature, called nucleus sampling, where the model considers the results of the tokens with top_p probability mass. So 0.1 means only the tokens comprising the top 10% probability mass are considered. We generally recommend altering this or temperature but not both. | 1.0     | no       | yes      |
 | maxTokens     | decimal | The maximum number of tokens to generate in the completion.                                                                                                                                                                                                                                                                            | 1000    | no       | yes      |
 
-
 ## Items Configuration
 
 Items to be used by the HLI service must be tagged with the [ "ChatGPT" ] tag.
 If no semantic model is set up, you can set the parameter `useSemanticModel` to false.
-In this case, the item names must follow the naming convention '<Location>_***', for example "Kitchen_Light". The label of the items are expected to briefly describe the item in more detail.
+In this case, the item names must follow the naming convention '{Location}_***', for example "Kitchen_Light". The label of the items are expected to briefly describe the item in more detail.
```

Comment:
Probably the `<>` syntax is preferable here, can it be restored in a rule-compliant way?

Reply:
Done